runtime/common/src/main/java/edu/snu/nemo/runtime/common/state/StageState.java

```diff
@@ -31,31 +35,17 @@ private StateMachine buildTaskStateMachine() {
     final StateMachine.Builder stateMachineBuilder = StateMachine.newBuilder();
 
     // Add states
-    stateMachineBuilder.addState(State.READY, "The stage has been created.");
-    stateMachineBuilder.addState(State.EXECUTING, "The stage is executing.");
+    stateMachineBuilder.addState(State.SCHEDULABLE, "This stage should be scheduled.");
```

Comment:
We may update the state description. 'Scheduling a stage' is vague in this two-state scheme, since it will still stay on `SCHEDULABLE` even after the scheduler schedules the stage.

One suggestion:
```
Some tasks in this stage is not completed.
```

Reply:
Or we may want to change `State.SCHEDULABLE` to `State.INCOMPLETE`.